doc/htmldoc/synapses/connection_management.rst

```diff
@@ -1,980 +1,547 @@
+		.. image:: ../static/img/All_to_all_H.png
 
-Multapses and autapses
-----------------------
+.. grid:: 4
+    :gutter: 1
 
-In the connection specification dictionary (containing the rule name and rule-
-specific parameters), the additional switch ``allow_autapses`` (default:
-``True``) can be set to allow or disallow self-connections.
+    .. grid-item-card::
+		:link: pairwise_bernoulli
+		:link-type: ref
 
-Likewise, ``allow_multapses`` (default: ``True``) can be used to specify if
-multiple connections between the same pair of neurons are allowed or not.
-
-These switches are only effective during each single call to
-:py:func:`.Connect`. Calling the function multiple times with the same set of
-neurons might still lead to violations of these constraints, even though the
-switches were set to `False` in each individual call.
-
-.. _conn_rules:
+		.. image:: ../static/img/Pairwise_bernoulli_H.png
 
-Connection Rules
-----------------
 
-Connection rules are specified using the ``conn_spec`` parameter of
-:py:func:`.Connect`, which can be either just a string naming a connection
-rule, or a dictionary containing a rule specification. Only connection
-rules requiring no parameters can be given as strings, for all other
-rules, a dictionary specifying the rule and its parameters is
-required. Examples for such parameters might be in- and out-degrees,
-or the probability for establishing a connection. A description of
-all connection rules available in NEST is available below.
+    .. grid-item-card::
+		:link: fixed_total_number
+		:link-type: ref
 
-all-to-all
-~~~~~~~~~~
+		.. image:: ../static/img/Fixed_total_number_H.png
+			
 
-.. image:: ../static/img/All_to_all.png
-     :width: 200px
-     :align: center
+    .. grid-item-card::
+		:link: fixed_indegree
+		:link-type: ref
 
-Each node in ``A`` is connected to every node in ``B``. Since
-``all_to_all`` is the default, the rule doesn't actually have to be
-specified.
+		.. image:: ../static/img/Fixed_indegree_H.png
 
-.. code-block:: python
+    .. grid-item-card::
+		:link: fixed_outdegree
+		:link-type: ref
 
-    n, m = 10, 12
-    A = nest.Create('iaf_psc_alpha', n)
-    B = nest.Create('iaf_psc_alpha', m)
-    nest.Connect(A, B, 'all_to_all')
-    nest.Connect(A, B)  # equivalent
+		.. image:: ../static/img/Fixed_outdegree_H.png
 
-.. _conn_builder_conngen:
+.. card::
+    :shadow: none
+    :text-align: center
 
-conngen
-~~~~~~~
+    Basic connection rules commonly used in the computational neuroscience community. For more details, go to the section :ref:`conn_rules` or just click on one of the illustrations.
 
-.. admonition:: Availability
+This documentation not only describes how to define network connectivity in NEST, but also represents a living reference for the connection rules defined in the article "Connectivity concepts in neuronal network modeling" [1]_. Deviations from the rules defined in the publication are highlighted in green.
```

Comment:
@heplesser Shall we add a statement suggesting that the paper could be cited as a permanent reference for a certain rule?

Reply:
Yes, that seems like a good idea!